src/sentry/tasks/reports.py

```diff
@@ -856,13 +904,17 @@ def summarize(key, points):
 
 def to_context(organization, interval, reports):
     report = reduce(merge_reports, reports.values())
-    series = [(to_datetime(timestamp), Point(*values)) for timestamp, values in report.series]
+    error_series = [
+        # Drop the transaction count from each series entry
+        (to_datetime(timestamp), Point(*values[:2]))
+        for timestamp, values in report.series
+    ]
     return {
-        "series": {
-            "points": series,
-            "maximum": max(sum(point) for timestamp, point in series),
-            "all": sum([sum(point) for timestamp, point in series]),
-            "resolved": sum([point.resolved for timestamp, point in series]),
+        "error_series": {
```

Comment:
Renamed this to `error_series` since it only represents the series of unresolved / resolved errors

Reply:
So if we want to make a `transactdion_series`, we just need to
```py
    transaction_series = [
        # Drop the transaction count from each series entry
        (to_datetime(timestamp), values[2])
        for timestamp, values in report.series
    ]
```
and then
```py
        "transaction_series": {
            "points": transaction_series,
            "maximum": max(point for timestamp, point in transaction_series),
            "all": sum(point for timestamp, point in transaction_series),
        },
```
right?